k8s/02_efs.sh

```diff
@@ -55,9 +55,9 @@ terraform init -backend-config=bucket=$S3_BUCKET \
 
 terraform apply -var aws_region=$AWS_REGION -var fs_subnet_id_zone_a=$subnetIdZoneA -var fs_subnet_id_zone_b=$subnetIdZoneB -var fs_sg_id=$securityGroupId -auto-approve
 
-export EFS_DNSNAME=`terraform output dns_name`
+export EFS_DNSNAME=`terraform output -raw dns_name`
```

Comment:
This made me think, could we document a terraform version number that we know the setup works with? Somewhere in README for example. WDYT?

Reply:
Yeah, that's a good idea. I'll see where can we have that documented. This is for the terraform v1.1.9.

Reply:
Ok, added a `README.md` file outlining this dependency inside the `k8s` folder